core/tests/protractor_desktop/profileFeatures.js

```diff
@@ -159,6 +159,14 @@ describe('Visiting user profile page', function() {
     profilePage.expectToHaveExplorationCardByName(EXPLORATION.title);
   });
 
+  it('should show created exploration stats for user', function() {
```

Comment:
There are other stats too right? Can we add tests for them all?

Reply:
I don't think the setup tooling is there for edit exploration or increase impact in the workflow helper yet, so I don't want to start building out new infra while the PR is already open or this PR might get in a state where it's open for a long time. We can add those to the tests workitem and I'll work on it in a separate effort

Reply:
Sure sg!